testing/environments/docker/kafka/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM spotify/kafka
```

Comment:
Lets do that in a follow up PR. We can do that now easily as the builds do not automatically start all images from the testing environment anymore, but each beat picks the once he needs. Like this we could also share kafka setup between metricbeat and libbeat.